src/Blog/View/post.twig

```diff
@@ -1,21 +1,55 @@
+
+Phasellus sem magna, tempus ut odio aliquam, consequat volutpat metus. <br> <br> Lorem ipsum dolor sit amet, consectetur adipiscing elit. Donec venenatis mauris varius erat dignissim, a ornare risus cursus. Phasellus mollis, turpis sit amet vulputate semper, metus felis dictum nunc, vitae commodo orci nisi nec enim. Curabitur consectetur dolor cursus velit tristique, eu cursus massa dapibus. Nam dignissim, justo semper porta venenatis, ante risus ultricies ipsum, sed efficitur velit ante eget massa. Praesent aliquam laoreet purus fermentum malesuada. In nec leo vel dui fermentum congue rhoncus id mi. Aenean ante sem, eleifend ullamcorper iaculis sit amet, fringilla hendrerit lectus. Vestibulum varius gravida odio eget sagittis. Nullam accumsan laoreet vulputate.
+
+Duis ultrices, velit interdum interdum sagittis, orci enim pulvinar sem, at congue enim velit eget lectus.  <br> <br> Nullam quis venenatis nisi. Phasellus scelerisque nisl nec odio sollicitudin bibendum. Duis ante sem, suscipit quis neque ac, viverra lobortis massa. Orci varius natoque penatibus et magnis dis parturient montes, nascetur ridiculus mus. Aliquam lacinia blandit orci, et euismod turpis lobortis nec. Nulla molestie eros vitae urna luctus, vel fringilla augue efficitur. Mauris ac nisi sagittis, posuere nisi quis, varius lectus. Vestibulum sagittis maximus dui ac accumsan. Etiam eros odio, dignissim eget ipsum id, ultricies commodo massa. Morbi mattis fringilla auctor. Nullam suscipit ante purus, vitae blandit neque vehicula ut. Pellentesque viverra nulla a leo laoreet rutrum. Proin venenatis, mauris et ultrices pulvinar, nibh dolor scelerisque tortor, nec aliquet elit enim vel nulla.
+
+Nunc venenatis lacus ac ex mattis efficitur. Sed nec quam ut nisl auctor commodo varius vel purus. Pellentesque facilisis tristique leo, eget egestas mi lobortis suscipit. Fusce lacinia magna interdum, fringilla orci sed, auctor augue. Quisque ultricies nibh vel massa feugiat, et dictum massa luctus. Curabitur id iaculis ligula, in mattis augue. Aliquam erat volutpat. Cras eu urna semper, varius ante ullamcorper, pulvinar leo. ',
     'url':'/img/slider1.svg',
     'date':'10-02-2020',
     'author_forename': 'John',
-    'author_surname': 'Nowak'
+    'author_surname': 'Nowak',
+	'author_position': 'Frontend Developer',
 }%}
 {% include 'Common/View/header.twig' %}
 {% block body %}
 	<body class="top-wave">
 	{% endblock %}
 	<section class="container">
-		<h1>{{ post.title }}</h1>
-		<p>{{ post.date }}</p>
-		<p>{{ post.author_forename }}
-			{{ post.author_surname }}</p>
-		<p>{{ post.content|raw }}</p>
+		<img src="/img/BlogPostHeader.jpg" alt="Blog Post Header" class="post-page__img">
+
+		<article class="u-margin--medium"> 
+			<div class=".post-page__header clearfix"> 
```

Comment:
The dots associate a class selector in the css file with the "class" attribute in HTML. So in this class name ".post-page__header", the period changes the class name, and the selector cannot find the item referenced by that class name. In the css file, the `.post-page__header {}` selector looks for a field with the class name `post-page__header`. 

I assume this was a small oversight, but I think it's better to keep that in mind ;)